Dockerfile.epp

```diff
@@ -18,8 +22,15 @@ COPY go.sum go.sum
 # Copy the go source
 COPY cmd/ cmd/
 COPY pkg/ pkg/
+COPY scripts/fetch-python-wrapper.sh scripts/fetch-python-wrapper.sh
+
+# Fetch requirements only (install in runtime)
+RUN curl -L https://raw.githubusercontent.com/llm-d/llm-d-kv-cache-manager/${KVCACHE_MANAGER_VERSION}/pkg/preprocessing/chat_completions/requirements.txt -o /workspace/requirements.txt
```

Comment:
we should do better with dependencies management. 
not a blocker, but we should probably handle that in the next iterations.